src/pages/[hubId]/[slug]/about.tsx

```diff
@@ -25,9 +25,13 @@ export const getStaticProps = getCommonStaticProps<AppCommonProps>(
 
     try {
       const [chat] = await getPostsServer([chatId])
-      title = chat.content?.title ?? ''
-      description = chat.content?.body ?? ''
-      image = chat.content?.image ? getIpfsContentUrl(chat.content?.image) : ''
+      if (!chat.struct.hidden) {
+        title = chat.content?.title ?? ''
+        description = chat.content?.body ?? ''
+        image = chat.content?.image
+          ? getIpfsContentUrl(chat.content?.image)
+          : ''
+      }
```

Comment:
Updated to at least become `content?.title` etc. I can't destructure it directly because they have same name with the `title` `description` `image` variable